zookeeper/settings.sls

```diff
@@ -67,54 +67,50 @@
 {%- set hosts_target         = g.get('hosts_target', p.get('hosts_target', 'roles:zookeeper')) %}
 {%- set targeting_method     = g.get('targeting_method', p.get('targeting_method', 'grain')) %}
 
-{%- set force_mine_update    = salt['mine.send'](hosts_function) %}
-{%- set zookeepers_host_dict = salt['mine.get'](hosts_target, hosts_function, targeting_method) %}
-{%- set zookeepers_ids       = zookeepers_host_dict.keys() %}
-{%- set zookeepers_hosts     = zookeepers_host_dict.values() %}
-
-{%- set zookeeper_host_num   = zookeepers_ids | length() %}
+{%- set zookeepers_with_ids       = [] %}
+{%- set zookeepers       = [] %}
+{%- set myid_dist = [] %}
+{%- set minion_ips = salt['network.ip_addrs']() %}
+{%- set connection_string = [] %}
```

Comment:
This is a minor item, but could you please align variables and values in columns separated by `=`? It would make statements much more readable. That's easy to accomplish with VIM and Tabular: http://vimcasts.org/episodes/aligning-text-with-tabular-vim/